cpp/ql/test/library-tests/dataflow/dataflow-tests/test-source-sink.expected

```diff
@@ -314,7 +318,11 @@ irFlow
 | test.cpp:1021:18:1021:32 | *call to indirect_source | test.cpp:1031:19:1031:28 | *translated |
 | test.cpp:1045:14:1045:19 | call to source | test.cpp:1046:7:1046:10 | * ... |
 | test.cpp:1052:13:1052:27 | *call to indirect_source | test.cpp:1054:7:1054:11 | * ... |
-| test.cpp:1089:27:1089:34 | call to source | test.cpp:1089:27:1089:34 | call to source |
+| test.cpp:1066:9:1066:14 | call to source | test.cpp:1072:10:1072:10 | i |
+| test.cpp:1066:9:1066:14 | call to source | test.cpp:1079:10:1079:10 | i |
+| test.cpp:1069:9:1069:14 | call to source | test.cpp:1074:10:1074:10 | i |
+| test.cpp:1069:9:1069:14 | call to source | test.cpp:1081:10:1081:10 | i |
```

Comment:
I don't know why 1066 and 1069 show up twice here. The other `source` invocations seem to only produce one result.

Reply:
This part of the file is showing edges - for example 1066 is showing up twice because there are two different edges out of `s2.i` (on line 1066), to two different [direct] uses of `s2.i` (on lines 1072, 1079).  You can see the same thing in some other test cases (e.g. `test.cpp` line 6).